asv_bench/benchmarks/utils.py

```diff
@@ -138,19 +140,19 @@ def gen_int_data(nrows: int, ncols: int, rand_low: int, rand_high: int) -> dict:
 
     Parameters
     ----------
-    nrows: int
+    nrows : int
         number of rows
-    ncols: int
+    ncols : int
         number of columns
-    rand_low: int
+    rand_low : int
         low bound for random generator
-    rand_high:int
+    rand_high : int
         high bound for random generator
 
-    Return
-    ------
+    Returns
+    -------
     dict
-        ncols number of keys, storing numpy.array of nrows length
+        Number of keys - `ncols`, each of them store numpy.array of `nrows` length.
```

Comment:
```suggestion
        Number of keys - `ncols`, each of them store np.ndarray of `nrows` length.
```

Reply:
We should use either `np.ndarray` or `NumPy array` because `numpy.array` doesn't exist.